deploy/olm-catalog/argocd-operator/0.2.0/argocd-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -897,6 +897,18 @@ spec:
           - ingresses
           verbs:
           - '*'
+        - apiGroups:
```

Comment:
At this point in time, we shouldn't be updating the 0.2.0 version of the CSV, since 0.2.0 has already shipped.

Reply:
agree, I will remove this from v0.2.0 and update the v0.3.0 manifests.

Reply:
removed the entries from v0.2.0 folder. @jopit PTAL